Roadmap.md

```diff
@@ -84,6 +85,22 @@
      - ✅ council keleres faction
  - ✅ Strategy cards (speaker handling etc)
  - ❌ Real names for things (currently we only use the ID for e.g objectives)
```

Comment:
Dessa är TODO mest för att komma ihåg att kolla på dem, flera av dessa är klara (åtminstonde i BE)